daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -364,6 +364,15 @@
             </xs:documentation>
           </xs:annotation>
         </xs:element>
+        <xs:element name="maxLookaheadFunctionBits" type="xs:long" default="512" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>
+              Max distance that the DPath lookahead function is permitted to look.
+              Distance is defined by the distance to the last bit accessed, and
+              so it is offset+bitsize.
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
```

Comment:
All of the tuanbles in dafext.xsd is sorted alphabetically. Can you move this one to the right spot? Then please squash, force commit, and then we can merge this. Thanks! :+1:

Reply:
Fixed and rebased